target/dita/regions/regions.dita

```diff
@@ -8,41 +8,33 @@
   <imagemap>
    <image href="./Content/Images/world-map.gif"/>
    <area>
-    <shape>
-     poly
-    </shape>
+    <shape>poly</shape>
```

Comment:
aah, @AbegaM - it's the `prettyfy` that is mangling this.
https://github.com/DeepBlueCLtd/LegacyMan/pull/268/files#diff-a6c6b35454c4bcb3844af5bc2c94deb5fd832767ce4eca996361fb50043a4121R105

Let's think about how to resolve it tomorrow.  I _guess_ we could run prettify, then re-process the tree to remove newlines from `<shape>` elements, then write that.  We may end up doing that a lot - so it should be a method that receives the type of element to _Clean-Up_.

Reply:
@IanMayo I have pushed a commit, instead of using `prettify` i have used the `xml.dom.minidom` library to prettify the code, can you run the code and check the `regions.dita`, please.